### ISSUE TITLE
Error with Homebrew install

### ISSUE THREAD

**Issue by dylanscott** [datawire/telepresence#275]
I previously was able to install telepresence via homebrew without issue, but now when try I get the following output:

```sh
$ brew install datawire/blackbird/telepresence
==> Installing telepresence from datawire/blackbird
Error: datawire/blackbird/telepresence contains conflicting version recursive dependencies:
  openssl, openssl@1.1
View these with `brew deps --tree datawire/blackbird/telepresence`.
```

Running the recommended deps command yields:

```sh
$ brew deps --tree datawire/blackbird/telepresence
datawire/blackbird/telepresence (required dependencies)
├── python3
│   ├── pkg-config
│   ├── sphinx-doc
│   ├── openssl
│   │   └── makedepend
│   │       └── pkg-config
│   ├── readline
│   ├── sqlite
│   │   └── readline
│   ├── gdbm
│   └── xz
├── torsocks
│   ├── :git
│   ├── autoconf
│   ├── automake
│   │   └── autoconf
│   └── libtool
├── sshfs
│   ├── pkg-config
│   ├── autoconf
│   ├── automake
│   │   └── autoconf
│   ├── libtool
│   └── glib
│       ├── pkg-config
│       ├── autoconf
│       ├── automake
│       │   └── autoconf
│       ├── libtool
│       ├── gettext
│       ├── libffi
│       └── pcre
└── socat
    ├── openssl@1.1
    └── readline
```

It looks like the [socat formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/socat.rb) was recently updated to depend on `openssl@1.1`, which conflicts with python3's `openssl` dependency.

I don't know enough about homebrew to know what can be done about this.

**Comment by dylanscott** [comment]
Also apologies if this would have made more sense to file on the [datawire/homebrew-blackbird](https://github.com/datawire/homebrew-blackbird) repo

**Comment by ark3** [comment]
Thanks for chasing this down. I released 0.65 that removes the `socat` dependency to work around this.
